ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -887,15 +887,26 @@ let vdi_copy_fun __context dbg vdi_map remote is_intra_pool remote_vdis so_far
     else
       (not is_intra_pool) || dest_sr <> vconf.sr
   in
+  let vbds = Db.VDI.get_VBDs ~__context ~self:vconf.vdi in
+  let _vm =
+    match vbds with
+    | [vbd] ->
+        Db.VBD.get_VM ~__context ~self:vbd
+    | _ ->
+        Ref.of_string ""
```

Comment:
This looks like an error case - should this be logged? The result value looks like it is going to confuse the caller. It might be better to use error handling.

Reply:
That makes sense, I am not sure if this will ever happen or the importance of the vm paramter in this contxt.